05week/higherOrder.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+// Create a forEach() function that takes an array of items and a function that
+// runs the function arr.length number of times.
+
+const myForEach = (arr) => {
+  const arrLength = () => {
```

Comment:
your forEach needs to also take in a callback, every item in your array should be passed to your callback. similar to your map below